src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts

```diff
@@ -728,6 +728,11 @@ export class ReadingsGraphComponent implements OnDestroy {
         }
       }
     };
+    if(this.optedTime > 86400){
+      this.assetChartOptions.scales.xAxes[0].time.unit = 'hour';
+      this.assetChartOptions.scales.xAxes[0].time.displayFormats.unit = 'hour';
+      this.assetChartOptions.scales.xAxes[0].time.displayFormats.hour = 'ddd HH:mm:ss';
```

Comment:
remove `:ss` at this scale.

:SSS can be removed for 8 hours & 1 Day data.

note: We always show that in tooltip of datapoint hover!

Reply:
`:ss` removed for 'hour' unit 
we are already not showing `:SSS` in any unit (showing it for tooltip only)